lib/widget/game.dart

```diff
@@ -337,8 +337,10 @@ class GameState extends State<Game> {
       ),
     ]..addAll(List.generate(playersSoFar.length, (i) => Text(playersSoFar[i].name)));
     return Center(
-      child: Card(
-        elevation: 2.0,
+      child: TitledCard(
+        title: AppLocalizations.of(context).room +
```

Comment:
We could have a string with the room code as a param to avoid assuming what the word order is, but we've done it like this in other places.

Reply:
I realised we did it properly in the other places so I've done it properly here too.